app/assets/images/boulder.svg

```diff
@@ -0,0 +1 @@
7.32" height="11.55"/></g><g class="cls-22"><rect class="cls-2" x="398.45" y="301.48" width="7.32" height="11.55"/></g><g class="cls-22"><rect class="cls-2" x="398.45" y="322.48" width="7.32" height="11.55"/></g><rect class="cls-10" x="615.48" y="193.38" width="7.32" height="11.55"/><rect class="cls-10" x="615.11" y="213.03" width="7.32" height="11.55"/><rect class="cls-10" x="631.48" y="193.38" width="7.32" height="11.55"/><rect class="cls-10" x="631.11" y="213.03" width="7.32" height="11.55"/><rect class="cls-10" x="615.29" y="234.2" width="7.32" height="7.8"/><rect class="cls-10" x="631.29" y="234.2" width="7.32" height="7.8"/><polygon class="cls-8" points="678.43 241.62 613.41 241.62 607.16 255.42 683.9 255.42 678.43 241.62"/><rect class="cls-10" x="688.83" y="272.01" width="7.78" height="6.02"/><rect class="cls-10" x="688.37" y="290.39" width="7.78" height="6.02"/><rect class="cls-10" x="688.37" y="307.39" width="7.78" height="6.02"/><rect class="cls-10" x="688.37" y="325.39" width="7.78" height="6.02"/><rect class="cls-2" x="797.49" y="278.89" width="7.78" height="6.02"/><rect class="cls-2" x="813.11" y="278.63" width="7.78" height="6.02"/><rect class="cls-2" x="829.3" y="278.76" width="7.78" height="6.02"/><rect class="cls-2" x="845.93" y="278.5" width="7.78" height="6.02"/><rect class="cls-2" x="862.1" y="278.43" width="7.78" height="6.02"/><rect class="cls-2" x="877.79" y="278.66" width="7.78" height="6.02"/><rect class="cls-2" x="796.64" y="292.89" width="7.78" height="6.02"/><rect class="cls-2" x="812.27" y="292.63" width="7.78" height="6.02"/><rect class="cls-2" x="828.45" y="292.76" width="7.78" height="6.02"/><rect class="cls-2" x="845.08" y="292.5" width="7.78" height="6.02"/><rect class="cls-2" x="861.25" y="292.43" width="7.78" height="6.02"/><rect class="cls-2" x="876.95" y="292.66" width="7.78" height="6.02"/><rect class="cls-2" x="796.64" y="307.89" width="7.78" height="6.02"/><rect class="cls-2" x="812.27" y="307.63" width="7.78" height="6.02"/><rect class="cls-2" x="828.45" y="307.76" width="7.78" height="6.02"/><rect class="cls-2" x="845.08" y="307.5" width="7.78" height="6.02"/><rect class="cls-2" x="861.25" y="307.43" width="7.78" height="6.02"/><rect class="cls-2" x="876.95" y="307.66" width="7.78" height="6.02"/><rect class="cls-2" x="796.64" y="322.89" width="7.78" height="6.02"/><rect class="cls-2" x="812.27" y="322.63" width="7.78" height="6.02"/><rect class="cls-2" x="828.45" y="322.76" width="7.78" height="6.02"/><rect class="cls-2" x="845.08" y="322.5" width="7.78" height="6.02"/><rect class="cls-2" x="861.25" y="322.43" width="7.78" height="6.02"/><rect class="cls-2" x="876.95" y="322.66" width="7.78" height="6.02"/><rect class="cls-10" x="897.22" y="278.66" width="7.78" height="6.02"/><rect class="cls-10" x="896.37" y="292.66" width="7.78" height="6.02"/><rect class="cls-10" x="896.37" y="307.66" width="7.78" height="6.02"/><rect class="cls-10" x="896.37" y="322.66" width="7.78" height="6.02"/></g></g></g></svg>
```

Comment:
Did you generate this svg from an image file?

Reply:
I made it in illustrator and that lets you export as an SVG!